src/Event.php

```diff
@@ -88,6 +88,20 @@ class Event implements PingableInterface
      */
     protected $timezone;
 
+    /**
+     * Datetime or time since the task is evaluated and possibly executed only for display purposes.
+     *
+     * @var \DateTime|string|null
+     */
+    protected $disply_from = null;
```

Comment:
No need to add `display` prefix, in both cases.

Reply:
Done